### PR TITLE
Bug 4786/v5

### PR DESCRIPTION
**Pull request by inashivb**

### CODE REVIEW

File: src/detect-flowbits.c [hunk 1]
```diff
@@ -207,125 +170,86 @@ int DetectFlowbitMatch (DetectEngineThreadCtx *det_ctx, Packet *p,
...
             }
-        }
+            break;
     }
 
-    return 1;
+    if (or_list_size != 0) {
+        for (uint32_t i = 0; i < or_list_size; i++) {
+            int leq = VarNameStoreSetupAdd(rs_xbits_get_or_list_item(cd, i), VAR_TYPE_FLOW_BIT);
+            rs_xbits_set_or_list_id_at(cd, i, leq);
```
Comment (inashivb) [inline src/detect-flowbits.c:225]: @jasonish need a little help here.. Made all the args `const void *` to send over to rust and to keep compatibility with the C fn calls but I also want to modify certain fields in Rust on the struct.
Here, I get the error for discarding const qualifier bc I try to get it as mut on the Rust side to modify it.

**Comment by suricata-qa** [comment]
WARNING:
 
 | field | baseline | test | % |
|--|--|--|--| 
| | build_asan | | |
 

Pipeline 10911

**Comment by suricata-qa** [comment]
ERROR:
 

ERROR: QA failed on build_asan.
  

Pipeline 10911

File: src/detect-flowbits.c (hunk 1 above)
Comment (jasonish) [reply src/detect-flowbits.c:225]: I think the answer is to not pass them as `const`, which might require API changes elsewhere as well.

**Comment by inashivb** [comment]
Needs a lot more work.

**Pull request closed without merging**